cmd/fs-v1-metadata.go

```diff
@@ -375,6 +383,8 @@ func checkFormatSanityFSV2(fsPath string) error {
 // Check for sanity of FS format for a given version.
 func checkFormatSanityFS(fsPath string, fsFormatVersion string) (err error) {
 	switch fsFormatVersion {
+	case fsFormatV1:
```

Comment:
> There is no use of maintaining older sanity checking functions. 

> Ah yes you are right @krishnasrinivas we don't need it.

Could you explain what the technical reason behind it is? I was not part of the discussion alluded to above (or I do not remember), but an actual reason is much stronger than recalling just the conclusion of a discussion from memory.

It looks to me like sanity checking protects us from failing to migrate an older version in case some assumption is not satisfied (and that assumption is checked in the sanity checker function).

If the argument against the above is that the migration function will have to handle all possible cases correctly, then why have a post-migration sanity check at all (since the migration function took care of everything)?